src/components/shared/Svg.css

```diff
@@ -1,14 +1,17 @@
 .arrow,
-.folder,
-.domain,
-.file,
 .worker,
 .refresh,
 .shortcut,
 .add-button {
   fill: var(--theme-splitter-color);
 }
 
+.file,
+.folder,
+.domain {
+  background-color: var(--theme-splitter-color);
```

Comment:
Out of curiosity, why does this use `background-color` whereas before it used `fill`?

Reply:
Fill is used to fill in paths of SVG's that are loaded into a webpage whereas background color loads a color into the div.  

With a mask being used over the div, the background color is what shows through the mask.  Hope that helps @James300

Reply:
Thank you, that makes a lot of sense :+1: